site/2017/10/24/odm-1-2-0-and-1-1-7.rst

```diff
@@ -0,0 +1,81 @@
+Doctrine MongoDB ODM 1.2.0 and 1.1.7 Released
+=============================================
+
+We are happy to announce the immediate availability of Doctrine MongoDB ODM
+`1.2.0 <https://github.com/doctrine/mongodb-odm/releases/tag/1.2.0>`_ and
+`1.1.7 <https://github.com/doctrine/mongodb-odm/releases/tag/1.1.7>`_.
+
+MongoDB ODM 1.1.7
+-----------------
+
+Notable fixes may be found in the
+`changelog <https://github.com/doctrine/mongodb-odm/blob/1.1.x/CHANGELOG-1.1.md#117-2017-10-23>`_.
+A full list of issues and pull requests included in this release may be found in the
+`1.1.7 milestone <https://github.com/doctrine/mongodb-odm/issues?q=milestone%3A1.1.7>`_.
+
+What is new in 1.2.0?
+---------------------
+
+Doctrine MongoDB ODM 1.2.0 comes with the following new features:
+
+- The ``readOnly`` mapping option allows you to map immutable documents.
+- With ``slaveOkay`` being deprecated, you can now specify a ``readPreference``
+  for your documents. This ``readPreference`` will automatically be applied to
+  all queries run on the collection.
```

Comment:
Would it be better to say "all queries for those mapped documents", since there could be multiple collections involved (discriminators) or possibly only some queries on a collection (if some custom query also reads from that collection but bypasses the document mapping).

I assume this works irrespective of hydration, though, since the mapping would already be considered if you start a query builder on some document class.

Reply:
Correct, it works irrespective of hydration and it applies to the collection as long as it's fetched from the document manager via `getDocumentCollection`. I'll change it to your suggested wording.